### ISSUE TITLE
do we need the `ModelName` `Enum` and the `Temperature` enum? 

### ISSUE THREAD

**Issue by kjappelbaum** [lamalab-org/corral#16]
in `mat-agent-bench/agents/agent.py`

While we gain some more control, it also requires more maintenance. And to properly maintain control, we would need to sync frequently with `LiteLLM`. If they have some list of model names they maintain, we might be better off using that or not controlling the model names at all on our side

**Comment by kjappelbaum** [comment]
With the temperature enum, I am not sure how universal "low", "medium", etc. are - in the end, it depends a bit on the calibration - i.e., how "flat", the output distribution is.  

What was the original reason for introducing this `Enum`?

**Comment by n0w0f** [comment]
> What was the original reason for introducing this `Enum`?

@MrtinoRG 's thought behind `Enum` for temperature, https://github.com/lamalab-org/mat-agent-bench/pull/3#discussion_r1898703072

**Comment by MrtinoRG** [comment]
It was to fix the values, but it can be removed

**Comment by kjappelbaum** [comment]
> It was to fix the values, but it can be removed

I agree with the comment that this is something that isn't much explored (and might be one of the easiest things one can start with for training - https://www.notion.so/Implement-simple-training-approaches-170988a4c88a80dfadcdd766c1cc0541?pvs=4). But I am not sure if and how we would like to fix the values for that
